src/TreeNode.jsx

```diff
@@ -600,11 +601,11 @@ class TreeNode extends React.Component {
 
         })}
 
-        onDragEnter={this.onDragEnter}
-        onDragOver={this.onDragOver}
-        onDragLeave={this.onDragLeave}
-        onDrop={this.onDrop}
-        onDragEnd={this.onDragEnd}
+        onDragEnter={draggable && this.onDragEnter}
+        onDragOver={draggable && this.onDragOver}
+        onDragLeave={draggable && this.onDragLeave}
+        onDrop={draggable && this.onDrop}
```

Comment:
Why do we prevent dropping by draggable?

Reply:
Documentation states: "whether can drag treeNode. ", nothing to do with drop functionality, so why do we hide it?